src/main/scanLogic/scanRunners/analyzerManager.ts

```diff
@@ -148,6 +151,33 @@ export class AnalyzerManager {
         };
     }
 
+    private isTokenValidationEnabled(): string {
+        let xraySemver: semver.SemVer = new semver.SemVer(this._connectionManager.xrayVersion);
+        if (xraySemver.compare(DYNAMIC_TOKEN_VALIDATION_MIN_XRAY_VERSION) < 0) {
+            this._logManager.logMessage(
+                'You cannot use dynamic token validation feature on xray version ' +
+                    this._connectionManager.xrayVersion +
+                    ' as it requires xray version ' +
+                    DYNAMIC_TOKEN_VALIDATION_MIN_XRAY_VERSION,
+                'INFO'
+            );
+            return 'false';
+        }
+        if (Configuration.enableTokenValidation()) {
+            return 'true';
+        }
```

Comment:
This will force the attribute in the Configurations, no matter what the API returns. just making sure this is OK

Reply:
yes. It should be stronger than the API